client/extensions/wp-super-cache/controller.js

```diff
@@ -17,8 +17,8 @@ import WPSuperCache from './main';
 
 const controller = {
 
-	settings: function( context ) {
-		const siteId = getSiteFragment( context.path );
+	settings: function(context, next) {
+	    const siteId = getSiteFragment( context.path );
```

Comment:
It looks like it still needs some tweaks with tabs in a line after function declaration. It also omits spaces around function's params. There is one simple solution - prettier :)